merlin/models/tf/prediction/item_prediction.py

```diff
@@ -143,6 +143,8 @@ class ItemRetrievalScorer(Block):
     item_id_feature_name: str
         Name of the column containing the item ids
         Defaults to `item_id`
+    query_name: str = 'query'
```

Comment:
Could use a single line of explanation, same for `item_name`

Reply:
yes for sure.